.github/workflows/draft-release.yml

```diff
@@ -0,0 +1,34 @@
+# Create a draft release to preview changelog without submitting production releases
+name: Draft Release
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    tags:
+      - 'v.*(-alpha|-beta).*' # Tag push events, i.e. v1.0.0-alpha, v1.0.0-alpha.1 v1.0.1-beta
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      # To use this repository's private action, you must check out the repository
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Jirafy Changelog
+        id: changelog
+        uses: ./
+        with:
+          myToken: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
myToken should be more descriptive...I see there's a GITHUB_TOKEN further down in this file. Is there a naming conflict, and are the 2 instances used for different purposes?